code/datums/outfits/outfit_admin.dm

```diff
@@ -135,7 +135,7 @@
 	back = /obj/item/storage/backpack/satchel
 	gloves = /obj/item/clothing/gloves/white
 	shoes = /obj/item/clothing/shoes/laceup
-	l_ear = /obj/item/device/radio/headset/heads/hop
+	l_ear = /obj/item/device/radio/headset/ert/ccia
```

Comment:
This probably needs to be changed to "agent" headset or something.

Reply:
This was the headset that the cciaa used for a while, that was changed to be the hop one by accident or for whatever reason.